packages/edit-navigation/docs/user-documentation.md

```diff
@@ -0,0 +1,72 @@
+Note: this documentation is a work in progress.
+
+# Block-based menu editor
+
+The Block-based Navigation Editor brings the power of blocks to the Appearance > Navigation section in the WordPress Administration Screens allowing you use blocks to create menus.
+
+## How to use the editor
+
+The interface replicates the Post Editor experience, allowing you to use similar workflows like drag and drop.
+
+The editor provides access to all the menus created using the previous Menu screen in WordPress.
+
+New menus can also easily be created, providing options to easily create a menu from an existing page hierarchy, by copying an existing menu, or creating a blank menu.
```

Comment:
```suggestion
New menus can also easily be created:
* from an existing `Page` hierarchy
* by copying an existing menu, 
* by creating a blank menu.
```

Reply:
I've gone half and half on this. I don't like leaving out that the editor 'provides options', because otherwise the 'by copying an existing menu' part makes it sound like the user has to find the menu, copy it, and then paste it.

Lets avoid using backticks for words because this formats it like code, which I don't think is relevant for user docs.

I only used them for the `+` button because it makes it look more like the icon, but I expect there will be images in the final docs.